.changeset/wet-tips-yawn.md

```diff
@@ -0,0 +1,10 @@
+---
+'@backstage/plugin-scaffolder': minor
+'@backstage/plugin-scaffolder-backend': minor
```

Comment:
```suggestion
'@backstage/plugin-scaffolder': minor
```

The sample template isn't part of the published package, so no need for a changeset